Build.ps1

```diff
@@ -234,36 +328,36 @@ function PromptUserForNuGetApiKeyCredential {
         $DestinationPath
     )
 
-    $message = "Enter your NuGet API Key in the password field (or nothing, this isn't used yet in the preview)"
-    $nuGetApiKeyCred = Get-Credential -Message $message -UserName "ignored"
```

Comment:
Can you change this function to take a parameter which becomes the message/prompt?  That way it doesn't have to be so watered down.  So for the NuGet API key (which actually probably should be the PowerShell Gallery API key), I would call it with `-Prompt 'Enter your PowerShell Gallery API Key in the password field (or nothing, this isn't used yet in the preview)'`.  And you can prompt folks for cert info.